lib/dojos.js

```diff
@@ -126,7 +127,7 @@ var joiValidator = {
       mysqlUserId: Joi.any(),
       firstName: Joi.any(),
       lastName: Joi.any(),
-      roles: Joi.array().required(),
+      roles: Joi.array(),
```

Comment:
I think a role will be required because you can either be {basic-user} or {cdf-admin}